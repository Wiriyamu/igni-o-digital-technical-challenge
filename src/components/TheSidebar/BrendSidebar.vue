<template>
  <figure class="logo">
    <img src="@/assets/images/TheSidebar/igd-logo.svg" alt="logo" />
  </figure>
</template>

<script>
export default {
  name: 'BrendSidebar'
}
</script>

<style lang="scss" scoped>
.logo {
  display: flex;
  justify-content: center;
  align-items: center;
  margin: 30px 0;
}
</style>
