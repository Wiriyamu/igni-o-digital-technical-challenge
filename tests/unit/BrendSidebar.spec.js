import BrendSidebar from '@/components/TheSidebar/BrendSidebar'
import { mount } from '@vue/test-utils'

describe('BrendSidebar.vue', () => {
  it('Snapshot component', () => {
    const wrapper = mount(BrendSidebar)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
