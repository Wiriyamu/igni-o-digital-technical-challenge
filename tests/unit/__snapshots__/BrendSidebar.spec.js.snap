// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrendSidebar.vue Snapshot component 1`] = `<figure class="logo"><img src="" alt="Ignição digital"></figure>`;
